journal-letters/original/06-chapter-4.adoc

```diff
@@ -2802,14 +3066,24 @@ Although some of his advocates endeavor to make out that his intended
 visit was almost unanimously approved in the meeting which liberated him,
 yet the published account states that the numbers on each side were nearly balanced.
 
+[.signed-section-closing]
 In love, I remain thy friend,
 
+[.signed-section-signature]
 Jonathan Evans.
 
+--
+
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 From Margaret Crosfield to J. W.
 
+[.salutation]
 Dear Friend,
 
+[.signed-section-context-open]
```

Comment:
I thought this looked funny here, but I wasn't sure if you wanted me to switch it around with the [.salutaion] or not. :)